packages/react-storage/src/components/StorageImage/types.ts

```diff
@@ -36,6 +38,7 @@ export interface StorageImageProps extends Omit<ImageProps, 'src'> {
 
 type OmittedPropKey =
   | 'accessLevel'
+  | 'bucket'
```

Comment:
Why are we omitting `bucket` here?

Reply:
+1, think its related to my comment above.

Reply:
Good point. This was an oversight on my part

Reply:
Removed